operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java

```diff
@@ -801,6 +755,13 @@ public boolean keyCreated() {
         return renewalType.equals(RenewalType.CREATE);
     }
 
+    /**
+     * @return Returns true if the renewal or replacement was postponed
```

Comment:
```suggestion
     * @return true if the renewal or replacement was postponed
```

Reply:
this Javadoc comment lines up with the one above for `keyCreated`, I guess we are inconsistent about whether we have `Returns` in the `@return` line for javadoc.